pydicer/convert/HN_137_RTstruct_to_Niffti_all - PET.py

```diff
@@ -0,0 +1,109 @@
+###################### Read Me #######################
+# This file is to convert .dicom files into .nifti files for PET imaging data, which can work on all patients.
+# Below is an example about how to use it.
+# BTW, from pet_to_nifti_convert_new import convert_dicom_to_nifty is an important function for converting PET's nifti files.
+#############Coded by Shuchao Pang 13 NOV 2020########
+
+import sys
+import os
+sys.path.append('../../..')
+import SimpleITK as sitk
+import pydicom
+import glob
+
+#from platipy.dicom.rtstruct_to_nifti.convert import convert_rtstruct
+#from pet_to_nifti_convert import convert_dicom
+from pet_to_nifti_convert_new import convert_dicom_to_nifty
+
+#dicom_search_path = "D:/Data/TCIA----Head and Neck Cancer/Head-Neck-PET-CT" # Location where Dicom data is stored
+dicom_search_path = "D:/New_Data/HEAD-NECK-RADIOMICS-HN1" # HN-137
+data_path = 'D:/New_Data/000NewCodes/niffty-PET-HN-137-new' # Path where converted files should be placed
+
+
+data_objects = {}
+number_case = 0
+
+for root, dirs, files in os.walk(dicom_search_path, topdown=False):

```

Comment:
The logic in this and the following loop is what is provided in pydicer's pipeline, so best to integrate all of this into there.